packages/image_picker/image_picker/ios/Classes/FLTPHPickerResultPopulatingOperation.h

```diff
@@ -0,0 +1,31 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import <Foundation/Foundation.h>
+#import <PhotosUI/PhotosUI.h>
+
+#import "FLTImagePickerImageUtil.h"
+#import "FLTImagePickerMetaDataUtil.h"
+#import "FLTImagePickerPhotoAssetUtil.h"
+
+/*!
+ @class FLTPHPickerResultPopulatingOperation
+
+ @brief The FLTPHPickerResultPopulatingOperation class
+
+ @discussion    This class was implemented to handle saved image paths and populate the pathList
+ with the final result by using GetSavedPath type block.
+
+ @superclass SuperClass: NSOperation\n
+ @helps It helps FLTImagePickerPlugin class.
+ */
+@interface FLTPHPickerResultPopulatingOperation : NSOperation
```

Comment:
Sorry for the back and forth. Since we are not populating results now, we might want a different name.
How about: `FLTPHPickerSaveImageToPathOperation` ?